test/xUnit/csharp/test_CommandLineParser.cs

```diff
@@ -0,0 +1,1143 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
Wow, this is really great!

Reply:
I always thought it was a little parser. 😄